saas_config.toml

```diff
@@ -54,6 +54,6 @@ identity_email = ""
 
 [zendesk]
```

Comment:
Fixing names here, not necessary for our build pipeline but this needs to match if you're testing locally.